applications/ClearDotTax.md

```diff
@@ -0,0 +1,257 @@
+
+- [CoinTracker](https://www.cointracker.io/)
+- [Koinly](https://app.koinly.io/)
+- [Netrunner](https://www.netrunner.tax/) - on Solana
+
+However, the above tools don't have support for Polkadot/Kusama and their Parachains (except EVM based ones).
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Team Lead: Rajdeep (https://twitter.com/rajgoesout)
+  - Building in the Polkadot ecosystem since ~2.5 years - led engineering at [YieldBay](https://github.com/yield-bay)
+  - Web3 Researcher/Rust Engineer
+  - Prev built products to solve for blockchain information asymmetry in the Filecoin ecosystem
+- Product Engineer: 0xDeep (https://twitter.com/0xDeep)
+  - Rust/Full-Stack Engineer
+  - Smart contract Auditor
+  - Prev wrote systems to process real-time data at a HFT firm
+- Product Designer: Axshay (https://twitter.com/axshay613)
+  - Designer [@polynomialfi](https://twitter.com/polynomialfi)
+  - Prev worked at a web3 game studio
+
+### Contact
+
+- **Contact Name:** Rajdeep Bharati
+- **Contact Email:** rajdeep@findsignal.studio
+- **Website:** https://www.findsignal.studio
+
+### Legal Structure
+
+- **Registered Address:** 3 Fraser Street, \#05-25 Duo Tower, Singapore (189352)
+- **Registered Legal Entity:** Find Signal Studio PTE. LTD.
+
+### Team's experience
+
+We've been building within the Polkadot ecosystem since 2020:
+
+- https://twitter.com/YieldScan - Staking dashboard for Polkadot and Kusama
+  - funded by Web3 Foundation Grants
+  - did 9 figures in transaction volume
+- https://twitter.com/yield_bay - Yield farming hub for Dotsama
+  - funded by the Polkadot Treasury
+  - tracks over 159 farms, 17 protocols, across 9 parachains
+
+### Team Code Repos
+
+- https://github.com/yield-bay/bay-watcher
+- https://github.com/yieldscan/yieldscan-frontend
+- https://github.com/yield-bay/bay-xcm-autocompound
+- https://github.com/yield-bay/bay-web
+
+**GitHub accounts of all team members:**
+
+- https://github.com/rajgoesout (yieldbay alt: https://github.com/JackSparrowYB)
+- https://github.com/0xdeepmehta
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/rajgoesout
+- https://twitter.com/0xDeep
+- https://twitter.com/axshay613
+
+### Project Details
+
+#### Key Deliverables
+
+- ##### Backend API
+  To fetch users' transactions across Polkadot and Kusama relay chains; and all major parachains of Polkadot & Kusama (ones supported by Subscan).
+- ##### [Transactions Dashboard](#milestone-1--transactions-dashboard)
+  To view and filter all transactions/income.
+- ##### Advanced transactions module
+
+  To track users' accrued income (fees/rewards) across the supported protocols:
+
+  1.  Lending Protocols<br>
+      a. Moonwell
+  2.  Staking Protocols<br>
+      a. Native DOT<br>
+      b. Manta<br>
+      c. Astar<br>
+      d. Hydradx<br>
+      e. Acala & Karura
+  3.  Yield Farms<br>
+      a. Stellaswap
+  4.  Liquidity Pools<br>
+      a. Stellaswap<br>
+      b. Acala & Karura
```

Comment:
please don't use any unclosed html tags. We need all applications to compile with mdx-5.
Give your application a check here. https://mdxjs.com/playground/